src/_includes/layouts/homepage.html

```diff
@@ -23,6 +26,7 @@ <h2 id="upcoming-events-section">Upcoming Events</h2>
           {% if event.data.type %}
           <span>{{ event.data.type }}</span>
           {% endif %}
+          <p>{{ event.data.date }}</p>
```

Comment:
Thank you so much for adding the date to the homepage - this will seriously help a lot! 😅